### PR TITLE
Latest service bot changes to 7.0.x

### PR DESCRIPTION
**Pull request by jainruchir**

### CODE REVIEW

File: .semaphore/cp_dockerfile_build.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ global_job_config:
     commands:
       - checkout
       - sem-version java 8
+      - sem-version python unknown
```
Comment (wadhwa1) [inline .semaphore/cp_dockerfile_build.yml:28]: not sure if this is correct

File: .semaphore/semaphore.yml [hunk 2]
```diff
@@ -25,6 +25,7 @@ global_job_config:
     commands:
       - checkout
       - sem-version java 8
+      - sem-version python unknown
```
Comment (wadhwa1) [inline .semaphore/semaphore.yml:28]: same here, not sure if adding `sem-version python unknown` will cause any issues?

File: sonar-project.properties [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ### service-bot sonarqube plugin managed file
 sonar.coverage.exclusions=**/test/**/*,**/tests/**/*,**/mock/**/*,**/mocks/**/*,**/*mock*,**/*test*
+sonar.cpd.exclusions=**/test/**/*,**/tests/**/*,**/mock/**/*,**/mocks/**/*,**/*mock*,**/*test*
```
Comment (wadhwa1) [inline sonar-project.properties:3]: is this also controlled by service bot?

Comment (wadhwa1) [reply sonar-project.properties:3]: looks like yes https://github.com/confluentinc/cc-service-bot/blame/ea44597089895ee4519075db4f4ae0613ce68fe2/sonar-project.properties#L4

File: .semaphore/cp_dockerfile_build.yml (hunk 1 above)
Comment (jainruchir) [reply .semaphore/cp_dockerfile_build.yml:28]: lets keep it same as master, i'll make this change

File: sonar-project.properties (hunk 3 above)
Comment (jainruchir) [reply sonar-project.properties:3]: i think we should remove it, cpd stuff is not needed

File: .semaphore/cp_dockerfile_build.yml (hunk 1 above)
Comment (wadhwa1) [reply .semaphore/cp_dockerfile_build.yml:28]: @jainruchir could you remove this line.. we can add it later.. for promotion, we don't need this anyway

File: .semaphore/semaphore.yml (hunk 2 above)
Comment (wadhwa1) [reply .semaphore/semaphore.yml:28]: @jainruchir could you remove this line.. we can add it later.. for promotion, we don't need this anyway

File: sonar-project.properties (hunk 3 above)
Comment (wadhwa1) [reply sonar-project.properties:3]: ok then

File: .semaphore/semaphore.yml (hunk 2 above)
Comment (jainruchir) [reply .semaphore/semaphore.yml]: yeah removed, master also does not set it

File: .semaphore/cp_dockerfile_build.yml (hunk 1 above)
Comment (jainruchir) [reply .semaphore/cp_dockerfile_build.yml]: removed

**Review by wadhwa1 — APPROVED** [review]

**Review by Anshul-Goyal — APPROVED** [review]

**Pull request merged**